task.go

```diff
@@ -60,7 +60,17 @@ func (t *Task) EffectiveCron() string {
 	return ""
 }
 
-// Run is a record created when a run of a task is scheduled.
+// LatestCompletedTime gives the time.Time that the task was last queued to be run at in RFC3339Nano format.
+func (t *Task) LatestCompletedTime() (time.Time, error) {
+	return time.Parse(time.RFC3339Nano, t.LatestCompleted)
```

Comment:
I believe this is a `time.RFC3339` it is set [here](https://github.com/influxdata/influxdb/blob/a17277f27ae0450d6b4565b149c21833c15e67f8/kv/task.go#L1570) based on the runs scheduledAt which is defined [here](https://github.com/influxdata/influxdb/blob/a17277f27ae0450d6b4565b149c21833c15e67f8/kv/task.go#L1289)